pulsar-broker/src/main/java/org/apache/pulsar/broker/web/ResponseHandlerFilter.java

```diff
@@ -55,8 +56,13 @@ public ResponseHandlerFilter(PulsarService pulsar) {
     public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
             throws IOException, ServletException {
 
+        if (!response.isCommitted()) {
+            ((HttpServletResponse) response).addHeader(BROKER_ADDRESS_HEADER_NAME, brokerAddress);
+        } else {
+            LOG.warn("Cannot add header {} to request {} since it's already committed.", BROKER_ADDRESS_HEADER_NAME,
+                    request);
```

Comment:
What's the actual output of the request#toString? 
Can we log just useful info, like client IP, METHOD and URI?

Reply:
yes it's possible. I'm simply assuming that the request.toString() method will provide something useful. :) This is something that isn't likely to happen since adding the header is now done before executing the chain.